app/controllers/index.js

```diff
@@ -6,7 +6,7 @@ export default Ember.Controller.extend({
   searchResults: null,
   count: 10,
 
-  instagramApiClient: null, // injected
+  instagramApiClient: Ember.inject.service(),
```

Comment:
You can inject the service anywhere in your app [this way](http://emberjs.com/blog/2015/02/07/ember-1-10-0-released.html#toc_injected-properties). No need for an initializer file anymore!